src/lib/utils/options/sources/typedoc.ts

```diff
@@ -21,6 +21,11 @@ export function addTypeDocOptions(options: Options) {
         type: ParameterType.Array
     });
 
+    options.addDeclaration({
```

Comment:
This implies that users can specify `--extends` on the command line, or in typedocOptions in tsconfig.json, which isn't true.

Reply:
Best to just remove it, I still need to update the website docs to mention this, but it shouldn't show up in --help.